conan/internal/api/audit/providers.py

```diff
@@ -169,24 +177,18 @@ def _parse_error(errors, ref):
 
         def _replace_message(message):
             if "not found" in message:
-                return f"{ref} was not found in the catalog"
+                return f"Package '{ref}' not scanned: Not found."
             return None
 
         error_msgs = filter(bool, [_replace_message(error["message"]) for error in errors])
-        return {"details": next(error_msgs, "Unknown error")}
+        return next(error_msgs, "Unknown error")
 
     def _get(self, ref):
         full_query = self._build_query(ref)
         try:
-            headers = {'Content-Type': 'application/json', 'Accept': 'application/json'}
-            if self.data.get("token"):
-                headers["Authorization"] = f"Bearer {self.data['token']}"
-            elif self.data.get("user") and self.data.get("password"):
-                headers["Authorization"] = f"Basic {self.data['user']}:{self.data['password']}"
```

Comment:
So no more user:password access?

Reply:
I think that was a leftover? It's not managed in the `conan audit provider` command, and also I think it's better to not have auth with user/password.

Reply:
> I think that was a leftover? It's not managed in the conan audit provider command, and also I think it's better to not have auth with user/password.

Yeah, I think so